src/OSPSuite.UI/Views/DisplayUnitsView.cs

```diff
@@ -80,17 +80,14 @@ public void BindTo(IEnumerable<DefaultUnitMapDTO> defaultUnits)
          _gridViewBinder.BindToSource(defaultUnits);
       }
 
-      public override bool HasError
-      {
-         get { return _gridViewBinder.HasError; }
-      }
+      public override bool HasError => _gridViewBinder.HasError;
 
       public override void InitializeResources()
       {
          base.InitializeResources();
-         layoutItemAddUnitMap.AdjustButtonSize(UIConstants.Size.BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
-         layoutItemLoadUnits.AdjustButtonSize(UIConstants.Size.BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
-         layoutItemSaveUnits.AdjustButtonSize(UIConstants.Size.BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
+         layoutItemAddUnitMap.AdjustTallButtonSize();
```

Comment:
Just created that method so that the knowledge of what is small, tall large etc. is only in one place